tests/storage-disks-vm

```diff
@@ -127,8 +127,8 @@ BLOCK_SPECIAL_FILE="$(lxc exec v2 -- readlink -f /dev/disk/by-id/scsi-0QEMU_QEMU
 lxc exec v2 -- test -b "$BLOCK_SPECIAL_FILE"
 lxc config device remove v2 ${LONG_DEVICE_NAME}
 sleep 3
-! lxc exec v2 -- stat "$BLOCK_SPECIAL_FILE" || false
-! lxc exec v2 -- stat /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_device-----with- || false
+! lxc exec v2 -- test -e "$BLOCK_SPECIAL_FILE" || false
```

Comment:
This change isn't moving a test and so shouldnt be in a commit called "Move volume related tests to storage-volumes-vm"

Reply:
I agree. Please be aware that this is still a draft so there are still some improvements left to be done designwise, I will notify when this is ready for a full review.